README.md

```diff
@@ -21,26 +21,61 @@ This repository provides solutions for configuring [LaunchDarkly code references
-| `exclude` | A regular expression defining the files and directories which the flag parser should exclude. |  | no |
-| `projKey` | A LaunchDarkly project key. |  | yes |
-| `updateSequenceId` | An integer representing the order number of code reference updates. Used to version updates across concurrent executions of the flag parser. If not provided, data will always be updated. If provided, data will only be updated if the existing `updateSequenceId` is less than the new `updateSequenceId`. Examples: the time a `git push` was initiated, CI build number, the current unix timestamp. |  | no |
-| `repoHead` | The HEAD or ref to retrieve code references from. Should be provided if the `git push` was initiated on a non-master branch. | "master" | no |
-| `repoName` | Git repo name. Will be displayed in LaunchDarkly |  | yes |
-| `repoType` | The repo service provider. Used to correctly categorize repositories in the LaunchDarkly UI. Acceptable values: github\|bitbucket\|custom | "custom" | no |
-| `repoUrl` | The display url for the repository. If provided for a github or bitbucket repository, LaunchDarkly will attempt to automatically generate source code links. Example: `https://github.com/launchdarkly/git-flag-parser` | "" | no |
-| `commitUrlTemplate` | If provided, LaunchDarkly will attempt to generate links to your Git service provider per commit. Example: `https://github.com/launchdarkly/git-flag-parser/commit/${sha}`. Allowed template variables: `branchName`, `sha`. If `commitUrlTemplate` is not provided, but `repoUrl` is provided, LaunchDarkly will automatically generate links for github or bitbucket repo types. | "" | no |
-| `hunkUrlTemplate` | If provided, LaunchDarkly will attempt to generate links to your Git service provider per code reference. Example: `https://github.com/launchdarkly/git-flag-parser/blob/${sha}/${filePath}#L${lineNumber}`. Allowed template variables: `sha`, `filePath`, `lineNumber`. If `hunkUrlTemplate` is not provided, but `repoUrl` is provided, LaunchDarkly will automatically generate links for github or bitbucket repo types. | "" | no |
+macOS users should download the darwin release for their respective system architecture.
+
+The `git-flag-parser` requires [The Silver Searcher](https://github.com/ggreer/the_silver_searcher#installing) to be installed as a dependency, so make sure it has been installed and added to your system PATH before running the git-flag-parser.
+
+A number of command-line arguments are available to the parser, some optional, and some required. Command line arguments may be passed to the program in any order.
+
+### Required arguments
+
+| Option | Description |
+|-|-|
+| `accessToken` | LaunchDarkly [personal access token](https://docs.launchdarkly.com/docs/api-access-tokens) with writer-level access, or access to the `code-references` [custom role](https://docs.launchdarkly.com/v2.0/docs/custom-roles) resource |
```

Comment:
we should have a ticket to update https://docs.launchdarkly.com/v2.0/docs/custom-roles with the code refs resource identifier.

Reply:
https://app.clubhouse.io/launchdarkly/story/29783/update-custom-role-docs